scripts/bootstrap.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
-
+mv /tmp/start-sql-config.txt /etc/yum.repos.d/mysql-community.repo

```

Comment:
Again, you don't need this 2 step action. The file - mysq-community.repo can be found in /vagrant during provisioning.